test/index.html

```diff
@@ -5,11 +5,12 @@
 <link rel="stylesheet" href="../build/test.css">
 <style>
   .fixture-Box {
-    background: rgb(100, 150, 50);
-    background: rgb(100, 150, 50);
-    border: 1px solid black;
+    background: rgba(100, 150, 50, .75);
+    border: 1px solid #888;
+    font-family: sans-serif;
     margin-bottom: 10px;
```

Comment:
There is a -something on the container (although this workaround can have unexpected results sometimes).

Anyway I am sold, let's postpone this for when someone will ask about it :)